README.md

```diff
@@ -51,42 +51,97 @@ print(space_eval(space, best))
 
 ## Contributing 
 
-### Setup
-If you're a developer, clone this repository and install from source:
+### Setup (based on [this](https://scikit-learn.org/stable/developers/contributing.html#contributing-code))
+If you're a developer and wish to contribute, please follow these steps:
+1. Create an account on GitHub if you do not already have one.
+2. Fork the project repository: click on the ‘Fork’ button near the top
+   of the page. This creates a copy of the code under your account on
+   the GitHub user account. For more details on how to fork a repository
+   see [this guide](https://help.github.com/articles/fork-a-repo/).
+3. Clone your fork of the hyperopt repo from your GitHub account to your
+   local disk:
 
 ```bash
-$ git clone https://github.com/hyperopt/hyperopt.git
+$ git clone https://github.com/<github username>/hyperopt.git
 $ cd hyperopt
+```
+
+### 4. Create a virtual env (python 3.x) to contain the dependencies and activate it
+on Windows using virtualenv:  
+`$ python -m venv venv`  
+or with conda  
+`$ conda create --name hyperopt`  
+or Linux/Unix  
+`$ python3 -m venv venv`
 
-# Create a virtual env (python 3.x) to contain the dependencies and activate it
-$ python3 -m venv venv
+```bash
 $ source venv/bin/activate
+$ python setup.py &&  pip install -e
```

Comment:
`python setup.py` alone will do nothing, it needs a command. Could you try just with the `pip install -e .....` ? I think it's not needed and the `pip install ....` will be enough

Reply:
It's not, it needs this `python setup.py develop && pip install -e '.[MongoTrials, SparkTrials, ATPE, dev]'`

Reply:
I tested locally, pip install is enough

Reply:
> 
> 
> I tested locally, pip install is enough

Got it, trusting you on this as this doesn't work in Windows.